### DISCUSSION TITLE
LG's Auto Switching to Filmmaker Mode not working with MPV

### DISCUSSION THREAD

**Discussion by cest-la-v** [mpv-player/mpv#13960] [Q&A]
I enabled LG's `Auto Switching to Filmmaker Mode` and set the graphic mode to `Filmmaker Mode` when using fullscreen app. It works fine with apps like potplayer (or games), the graphic mode automatically switched `Filmmaker Mode` as expected when entering fulllscreen and switched back when exiting fulllscreen. But it won't work with MPV, I tried with `d3d11-exclusive-fs = yes`, no luck.

Anyone know is it a setting issue or is it MPV just don't support this? Thanks!

My setup:
- LG 48C3
- Windows 11

**Comment by natural-harmonia-gropius** [comment]
Try this

```
gpu-api=vulkan
ontop
```

**Comment by cest-la-v** [comment]
@natural-harmonia-gropius  Still didn't work. Never mind, I'll stick to switching manually for now. Thanks though 😶‍🌫️
